src/components/DataTypes/object.tsx

```diff
@@ -194,6 +195,15 @@ export const createFilter = (
 	schema: JSONSchema6,
 ): JSONSchema6 => {
 	const { title } = schema;
+
+	if (operator.includes('key')) {
```

Comment:
I have a bit of a hard time understanding the solution here, as it feels quite hacky to me. If you can't find a better approach, at the very least, I would extract this into a function that clearly says what this does.